# Sardinian translation for update-manager
# Copyright (c) 2009 Rosetta Contributors and Canonical Ltd 2009
# This file is distributed under the same license as the update-manager package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2009.
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2014-04-28 21:02+0000\n"
"PO-Revision-Date: 2010-09-06 03:55+0000\n"
"Last-Translator: Borealis <Unknown>\n"
"Language-Team: Sardinian <sc@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n!=1;\n"
"X-Launchpad-Export-Date: 2014-07-07 12:28+0000\n"
"X-Generator: Launchpad (build 17086)\n"
"Language: sc\n"

#: ../UpdateManager/backend/InstallBackendAptdaemon.py:67
msgid "Checking for updates…"
msgstr ""

#: ../UpdateManager/backend/InstallBackendAptdaemon.py:92
msgid "Installing updates…"
msgstr ""

#: ../UpdateManager/backend/InstallBackendSynaptic.py:60
msgid "Please wait, this can take some time."
msgstr ""

#: ../UpdateManager/backend/InstallBackendSynaptic.py:62
msgid "Update is complete"
msgstr ""

#: ../UpdateManager/ChangelogViewer.py:73
msgid "Open Link in Browser"
msgstr ""

#: ../UpdateManager/ChangelogViewer.py:76
msgid "Copy Link to Clipboard"
msgstr ""

#: ../UpdateManager/Dialogs.py:115
msgid "Settings…"
msgstr ""

#: ../UpdateManager/Dialogs.py:144 ../UpdateManager/UpdateManager.py:247
msgid "You stopped the check for updates."
msgstr ""

#: ../UpdateManager/Dialogs.py:146
msgid "_Check Again"
msgstr ""

#: ../UpdateManager/Dialogs.py:158
msgid "No software updates are available."
msgstr ""

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:388 ../UpdateManager/UpdateManager.py:680
msgid "The software on this computer is up to date."
msgstr ""

#. Translators: these are Ubuntu version names like "Ubuntu 12.04"
#: ../UpdateManager/Dialogs.py:172
#, python-format
msgid "However, %s %s is now available (you have %s)."
msgstr ""

#: ../UpdateManager/Dialogs.py:177
msgid "Upgrade…"
msgstr ""

#. Translators: this is an Ubuntu version name like "Ubuntu 12.04"
#: ../UpdateManager/Dialogs.py:200
#, python-format
msgid "Software updates are no longer provided for %s %s."
msgstr ""

#. Translators: this is an Ubuntu version name like "Ubuntu 12.04"
#: ../UpdateManager/Dialogs.py:204
#, python-format
msgid "To stay secure, you should upgrade to %s %s."
msgstr ""

#: ../UpdateManager/Dialogs.py:217
msgid "Not all updates can be installed"
msgstr ""

#: ../UpdateManager/Dialogs.py:219
msgid ""
"Run a partial upgrade, to install as many updates as possible.\n"
"\n"
"    This can be caused by:\n"
"     * A previous upgrade which didn't complete\n"
"     * Problems with some of the installed software\n"
"     * Unofficial software packages not provided by Ubuntu\n"
"     * Normal changes of a pre-release version of Ubuntu"
msgstr ""

#: ../data/gtkbuilder/UpdateManager.ui.h:3
msgid "_Partial Upgrade"
msgstr ""

#: ../DistUpgrade/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr ""

#: ../UpdateManager/Dialogs.py:265
msgid "_Try Again"
msgstr ""

#: ../UpdateManager/Dialogs.py:279
msgid "The computer needs to restart to finish installing updates."
msgstr ""

#: ../UpdateManager/Dialogs.py:281
msgid "Restart _Later"
msgstr ""

#: ../UpdateManager/Dialogs.py:282
msgid "_Restart Now…"
msgstr ""

#. Basic GTK+ parameters
#: ../UpdateManager/UpdateManager.py:87 ../data/update-manager.desktop.in.h:1
msgid "Software Updater"
msgstr ""

#: ../UpdateManager/UpdateManager.py:245
msgid "Some software couldn’t be checked for updates."
msgstr ""

#: ../UpdateManager/UpdateManager.py:248
msgid "Updated software is available from a previous check."
msgstr ""

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:1159
msgid "Software index is broken"
msgstr ""

#: ../UpdateManager/UpdateManager.py:1160
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""

#: ../UpdateManager/UpdateManager.py:1008
msgid "Could not initialize the package information"
msgstr ""

#: ../UpdateManager/UpdateManager.py:1009
msgid ""
"An unresolvable problem occurred while initializing the package "
"information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:697
#: ../UpdateManager/UpdateManager.py:1037
msgid "Could not calculate the upgrade"
msgstr ""

#: ../UpdateManager/UpdateManager.py:352
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""

#: ../UpdateManager/UpdatesAvailable.py:251
msgid "Install Now"
msgstr ""

#: ../UpdateManager/UpdateManager.py:231
#: ../UpdateManagerText/UpdateManagerText.py:32
msgid "Install"
msgstr ""

#: ../UpdateManager/UpdatesAvailable.py:318
msgid "Download"
msgstr ""

#: ../UpdateManager/UpdatesAvailable.py:377
msgid "_Remind Me Later"
msgstr ""

#. upload_archive = version_match.group(2).strip()
#: ../UpdateManager/UpdateManager.py:393
#, python-format
msgid "Version %s: \n"
msgstr ""

#: ../UpdateManager/UpdateManager.py:450
msgid ""
"No network connection detected, you can not download changelog information."
msgstr ""

#: ../UpdateManager/UpdateManager.py:458
msgid "Downloading list of changes..."
msgstr ""

#: ../UpdateManager/UpdateManager.py:502
msgid "_Deselect All"
msgstr ""

#: ../UpdateManager/UpdateManager.py:508
msgid "Select _All"
msgstr ""

#: ../UpdateManager/UpdateManager.py:572
#, python-format
msgid "%s will be downloaded."
msgstr ""

#: ../UpdateManager/UpdatesAvailable.py:746
msgid "The update has already been downloaded."
msgid_plural "The updates have already been downloaded."
msgstr[0] ""
msgstr[1] ""

#: ../UpdateManager/UpdateManager.py:589
msgid "There are no updates to install."
msgstr ""

#: ../UpdateManager/UpdateManager.py:602
msgid "Unknown download size."
msgstr ""

#: ../UpdateManager/UpdatesAvailable.py:788
#, python-format
msgid ""
"Updated software has been issued since %s %s was released. Do you want to "
"install it now?"
msgstr ""

#: ../UpdateManager/UpdatesAvailable.py:793
msgid ""
"Updated software is available for this computer. Do you want to install it "
"now?"
msgstr ""

#: ../UpdateManager/UpdatesAvailable.py:796
msgid ""
"The computer also needs to restart to finish installing previous updates."
msgstr ""

#. print "on_button_install_clicked"
#: ../DistUpgrade/DistUpgradeController.py:866
#: ../UpdateManager/UpdateManager.py:755
msgid "Not enough free disk space"
msgstr ""

#: ../UpdateManager/UpdateManager.py:756
#, python-format
msgid ""
"The upgrade needs a total of %s free space on disk '%s'. Please free at "
"least an additional %s of disk space on '%s'. Empty your trash and remove "
"temporary packages of former installations using 'sudo apt-get clean'."
msgstr ""

#: ../UpdateManager/UpdateManager.py:863
msgid "Connecting..."
msgstr ""

#: ../UpdateManager/UpdateManager.py:880
msgid "You may not be able to check for updates or download new updates."
msgstr ""

#: ../UpdateManager/UpdatesAvailable.py:1018
msgid "Security updates"
msgstr ""

#: ../UpdateManager/Core/UpdateList.py:70
msgid "Other updates"
msgstr ""

#: ../UpdateManager/UnitySupport.py:66
msgid "Install All Available Updates"
msgstr ""

#: ../UpdateManager/UpdateManager.py:1256
#: ../UpdateManagerText/UpdateManagerText.py:31
msgid "Cancel"
msgstr ""

#: ../UpdateManagerText/UpdateManagerText.py:34
msgid "Changelog"
msgstr ""

#: ../UpdateManagerText/UpdateManagerText.py:37
msgid "Updates"
msgstr ""

#: ../UpdateManagerText/UpdateManagerText.py:50
msgid "Building Updates List"
msgstr ""

#: ../UpdateManagerText/UpdateManagerText.py:57
msgid ""
"\n"
"A normal upgrade can not be calculated, please run:\n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""

#: ../UpdateManagerText/UpdateManagerText.py:122
msgid "Downloading changelog"
msgstr ""

#: ../UpdateManager/Core/MyCache.py:316
msgid "This update does not come from a source that supports changelogs."
msgstr ""

#: ../UpdateManager/Core/MyCache.py:322 ../UpdateManager/Core/MyCache.py:350
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr ""

#: ../UpdateManager/Core/MyCache.py:346
#, python-format
msgid ""
"Changes for %s versions:\n"
"Installed version: %s\n"
"Available version: %s\n"
"\n"
msgstr ""

#: ../UpdateManager/Core/MyCache.py:339
#, python-format
msgid ""
"The changelog does not contain any relevant changes.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""

#: ../UpdateManager/Core/MyCache.py:344
#, python-format
msgid ""
"The list of changes is not available yet.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""

#. Translators: the %s is a distro name, like 'Ubuntu' and 'base' as in
#. the core components and packages.
#: ../UpdateManager/Core/UpdateList.py:167
#, python-format
msgid "%s base"
msgstr ""

#. TRANSLATORS: download size of small updates, e.g. "250 kB"
#: ../DistUpgrade/utils.py:399 ../UpdateManager/Core/utils.py:399
#, python-format
msgid "%(size).0f kB"
msgid_plural "%(size).0f kB"
msgstr[0] ""
msgstr[1] ""

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:402 ../UpdateManager/Core/utils.py:402
#, python-format
msgid "%.1f MB"
msgstr "%.1f MiB"

#: ../data/gtkbuilder/UpdateManager.ui.h:28
msgid "updates"
msgstr ""

#: ../data/gtkbuilder/UpdateManager.ui.h:33
msgid "Changes"
msgstr ""

#: ../data/gtkbuilder/UpdateManager.ui.h:34
msgid "Description"
msgstr ""

#: ../data/gtkbuilder/UpdateManager.ui.h:4
msgid "Technical description"
msgstr ""

#: ../data/gtkbuilder/UpdateManager.ui.h:5
msgid "Details of updates"
msgstr ""

#: ../data/gtkbuilder/UpdateManager.ui.h:6
msgid "The computer will need to restart."
msgstr ""

#: ../data/gtkbuilder/UpdateManager.ui.h:29
msgid ""
"You are connected via roaming and may be charged for the data consumed by "
"this update."
msgstr ""

#: ../data/gtkbuilder/UpdateManager.ui.h:8
msgid ""
"You may want to wait until you’re not using a mobile broadband connection."
msgstr ""

#: ../data/gtkbuilder/UpdateManager.ui.h:31
msgid "It’s safer to connect the computer to AC power before updating."
msgstr ""

#: ../data/update-manager.desktop.in.h:2
msgid "Software Updates"
msgstr ""

#: ../data/update-manager.desktop.in.h:3
msgid "Show and install available updates"
msgstr ""

#: ../update-manager:61 ../update-manager-text:50 ../do-release-upgrade:46
msgid "Show version and exit"
msgstr ""

#: ../update-manager:64
msgid "Directory that contains the data files"
msgstr ""

#: ../update-manager:67
msgid "Check if a new Ubuntu release is available"
msgstr ""

#: ../update-manager:70 ../do-release-upgrade:49 ../check-new-release-gtk:182
msgid "Check if upgrading to the latest devel release is possible"
msgstr ""

#: ../update-manager:74
msgid "Upgrade using the latest proposed version of the release upgrader"
msgstr ""

#: ../update-manager:81
msgid "Do not focus on map when starting"
msgstr ""

#: ../update-manager:87
msgid "Do not check for updates when starting"
msgstr ""

#: ../update-manager:91 ../do-release-upgrade:65
msgid "Test upgrade with a sandbox aufs overlay"
msgstr ""

#: ../update-manager-text:54
msgid "Show description of the package instead of the changelog"
msgstr ""

#: ../ubuntu-support-status:79
msgid "Show unsupported packages on this machine"
msgstr ""

#: ../ubuntu-support-status:82
msgid "Show supported packages on this machine"
msgstr ""

#: ../ubuntu-support-status:85
msgid "Show all packages with their status"
msgstr ""

#: ../ubuntu-support-status:88
msgid "Show all packages in a list"
msgstr ""

#: ../ubuntu-support-status:130
#, c-format
msgid "Support status summary of '%s':"
msgstr ""

#: ../ubuntu-support-status:133
msgid "You have %(num)s packages (%(percent).1f%%) supported until %(time)s"
msgstr ""

#: ../ubuntu-support-status:139
msgid ""
"You have %(num)s packages (%(percent).1f%%) that can not/no-longer be "
"downloaded"
msgstr ""

#: ../ubuntu-support-status:142
msgid "You have %(num)s packages (%(percent).1f%%) that are unsupported"
msgstr ""

#: ../ubuntu-support-status:155
msgid ""
"Run with --show-unsupported, --show-supported or --show-all to see more "
"details"
msgstr ""

#: ../ubuntu-support-status:159
msgid "No longer downloadable:"
msgstr ""

#: ../ubuntu-support-status:162
msgid "Unsupported: "
msgstr ""

#: ../ubuntu-support-status:167
#, c-format
msgid "Supported until %s:"
msgstr ""

#: ../ubuntu-support-status:176
msgid "Unsupported"
msgstr ""

#: ../Janitor/computerjanitor/exc.py:30
#, python-format
msgid "Unimplemented method: %s"
msgstr ""

#: ../Janitor/computerjanitor/file_cruft.py:46
msgid "A file on disk"
msgstr ""

#: ../Janitor/computerjanitor/missing_package_cruft.py:33
msgid "Install missing package."
msgstr ""

#: ../Janitor/computerjanitor/missing_package_cruft.py:42
#, python-format
msgid "Package %s should be installed."
msgstr ""

#: ../Janitor/computerjanitor/package_cruft.py:45
msgid ".deb package"
msgstr ""

#: ../Janitor/plugins/langpack_manual_plugin.py:34
#, python-format
msgid "%s needs to be marked as manually installed."
msgstr ""

#: ../Janitor/plugins/kdelibs4to5_plugin.py:39
msgid ""
"When upgrading, if kdelibs4-dev is installed, kdelibs5-dev needs to be "
"installed. See bugs.launchpad.net, bug #279621 for details."
msgstr ""

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:36
#, python-format
msgid "%i obsolete entries in the status file"
msgstr ""

#: ../Janitor/plugins/dpkg_status_plugin.py:39
msgid "Obsolete entries in dpkg status"
msgstr ""

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:42
msgid "Obsolete dpkg status entries"
msgstr ""

#: ../Janitor/plugins/remove_lilo_plugin.py:28
msgid ""
"Remove lilo since grub is also installed.(See bug #314004 for details.)"
msgstr ""
